code/game/mecha/mecha.dm

```diff
@@ -1109,9 +1112,11 @@
 		return
 	lights = !lights
 	if(lights)
-		set_light(luminosity + lights_power)
+		light_power = light_brightness_on
```

Comment:
Try using this instead

```
light_power = lights ? light_brightness_on : light_brightness_off
set_light(light_power)
```

Reply:
Power and range are two different variables. Power affects the "intensity" of the light, ie. how brightly it illuminates tiles within its range, but does not extend the range. Range is the number of tiles the light extends outwards from its source. For a visual comparison, [this](https://user-images.githubusercontent.com/17928298/43481207-b47991fe-94db-11e8-9b0a-258ecc079e29.png) is range 6 power 1, and [this](https://user-images.githubusercontent.com/17928298/43481294-ef54d96e-94db-11e8-9c48-46ff976adb94.png) is range 6 power 2.

Reply:
I'm just dumb